repos/fdbt-site/src/pages/api/deletePassenger.ts

```diff
@@ -25,6 +28,30 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             throw new Error('This individual cannot be deleted because it is in use in a group.');
         }
 
+        const products = await getAllProductsByNoc(nationalOperatorCode)
+        const matchingJsonLinks = products.map((product) => product.matchingJsonLink);
+        const tickets = await Promise.all(
+            matchingJsonLinks?.map(async (link) => {
```

Comment:
do we require it like ?.
 we can use matchingJsonLinks.map instead

Reply:
if matchingJson comes out null .map will throw an error ?. prevents that